Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: test
+
+test: database-v1
+	@go test -v -race ./...
+
+database-v1:
```

Comment:
How about splitting this into two targets, one for the tarball and one for the directory?